frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -101,11 +101,7 @@ const ClusterAlerts = connectToFlags(FLAGS.OPENSHIFT)(
       const cvLoaded = _.get(resources.cv, 'loaded');
       const LinkComponent = React.useCallback(
         () => (
-          <Button
-            className="co-status-card__link-button"
```

Comment:
Looks like this class is no longer being used and should be removed from https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss#L97-L99